explore/src/main/scala/explore/config/GeneratedSequenceViewer.scala

```diff
@@ -28,16 +28,12 @@ object GeneratedSequenceViewer {
   val component =
     ScalaFnComponent
       .withHooks[Props]
-      .useStreamOnMountBy { props =>
+      .useEffectResultOnMountBy { props =>
```

Comment:
I wonder if we'd need a hook that would take deps and be reactivated if the deps change

Reply:
There's a version that does that: `useEffectResultWithDepsBy`